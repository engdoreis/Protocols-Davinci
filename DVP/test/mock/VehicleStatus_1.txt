WheelLock=1
poweredOn=0
headLightOn=0
tailLightOn=0
buzzerOn=0
cruiseOn=0
speed=0