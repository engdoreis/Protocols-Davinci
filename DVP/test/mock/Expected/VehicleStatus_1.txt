Ret=0
 WheelLock=0
 poweredOn=1
 headLightOn=1
 tailLightOn=1
 buzzerOn=0
 cruiseOn=0
 speed=3315